Application/EdFi.Ods.Admin.Api/E2E Tests/Admin API E2E.postman_collection.json

```diff
@@ -1154,10 +1151,9 @@
 											"pm.test(\"Response matches error format\", function () {\r",
 											"    const response = pm.response.json();\r",
 											"\r",
-											"    pm.expect(response.status).to.equal(404);\r",
+											"    pm.expect(response).to.have.property(\"status\");\r",
 											"    pm.expect(response).to.have.property(\"title\");\r",
-											"    pm.expect(response).to.have.property(\"errors\");\r",
-											"    pm.expect(response.errors).to.contain(response.title);\r",
+											"    pm.expect(response.status).to.equal(404);\r",
```

Comment:
I made a few edits while I was debugging that may not be strictly necessary. I added the check on `response.status` back to the validation after giving more thought to it.

A 404 shouldn't have a list of errors.